guides/installation/from-scratch.md

```diff
@@ -75,10 +137,6 @@ A simple cli installation wizard can be invoked by executing:
 bin/setup
 ```
 
-{% hint style="info" %}
-One little note: If something goes wrong during installation check if `.psh.yaml.override` exists. If not restart setup, if yes execute `./psh.phar install` to restart the setup process.
-{% endhint %}
-
 Voila, Shopware 6 is installed. To be sure the installation succeeded, just open the configured host url in your favorite browser.
 
 ## Updating the repositories
```

Comment:
@elkmod About the contents of the section below this heading, it seems additional review may be required:
@shyim said that the production repository is still the recommended option, but I haven't ever set it up, it seems `bin/setup` in that repo doesn't exist, for example

Reply:
We need a second guide for production setups anyway, so we can leave it as is.